### PR TITLE
Update 'ABIS GmbH' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.netlify.com/#!doc=%7B%22slug%22%3A%22abis%22%2C%22relevant-countries%22%3A%5B%22de%22%5D%2C%22categories%22%3A%5B%22addresses%22%5D%2C%22name%22%3A%22ABIS%20GmbH%22%2C%22address%22%3A%22Lyoner%20Str.%2020%5Cn60528%20Frankfurt%20am%20Main%5CnDeutschland%22%2C%22phone%22%3A%22%2B49%2069%207920090%22%2C%22fax%22%3A%22%2B49%2069%2079200920%22%2C%22email%22%3A%22datenschutz%40abis-online.de%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.abis-online.de%2Fdatenschutz%2Fdatenschutzanfrage%2F%22%2C%22https%3A%2F%2Fwww.abis-online.de%2Fkontakt%2Fimpressum%2F%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F812%22%5D%2C%22required-elements%22%3A%5B%7B%22desc%22%3A%22Name%22%2C%22type%22%3A%22name%22%7D%2C%7B%22desc%22%3A%22Adresse%22%2C%22type%22%3A%22address%22%7D%5D%2C%22suggested-transport-medium%22%3A%22fax%22%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

**Comment by mal-tee** [comment]
When I made a request via email in 2020-05 the wrote back "Gerne beantworten wir Ihnen Ihre Anfrage. Bitte haben Sie jedoch Verständnis dafür, dass wir eine Auskunft gem. Art. 15 DS-GVO nur dann erteilen können, wenn uns die im beigefügten Schreiben genannten Angaben in schriftlicher Form vorliegen.", so I guess `fax` makes sense?

**Comment by mal-tee** [comment]
I can also confirm the comment on the company that they want an "eigenhändige Unterschrift", so i guess we should make a comment in the record?

**Comment by baltpeter** [comment]
This seems to depend on whether they have data on you. I got the (written) confirmation that they don't have any data on me by simply sending an email (in September 2020), without having to submit any additional requirements.

As such, I don't think changing the record to 'fax' is a good idea. The signature isn't necessary in all cases, most users don't have access to a fax machine and according to the comment they also accept the signature via email.  
I think it's best if we simply add a comment explaining that to the record. Users probably won't know beforehand whether Abis has data on them, so it doesn't make sense to voluntarily submit a signature in all cases.

**Comment by baltpeter** [comment]
Only slightly related: It would be great to have a supervisory authority decide whether the practice of requiring the signature is even lawful. The GDPR doesn't specify any "Formerfordernisse" and a signature obviously cannot be used for identification purposes.

Unfortunately I cannot submit a complaint as they didn't want a signature for my request. :D

**Comment by rugk** [comment]
Ok, I'll do.

**Comment by rugk** [comment]
Sent. I'll keep you up-to-date.

**Comment by mal-tee** [comment]
Great! Thanks.

File: companies/abis.json [hunk 1]
```diff
@@ -22,10 +26,6 @@
             "type": "address"
         }
     ],
-    "relevant-countries": [
-        "de"
-    ],
-    "needs-id-document": false,
-    "suggested-transport-medium": "email",
-    "quality": "tested"
+    "suggested-transport-medium": "fax",
```
Comment (mal-tee) [inline companies/abis.json:29]:
```suggestion
    "comments": [
        "Falls die ABIS GmbH keine Daten über Dich gespeichert hat, wird sie Dir dies mitteilen, ansonsten will sie meist eine Unterschrift auf der Anfrage."
    ],
```

File: companies/abis.json [hunk 2]
```diff
@@ -22,10 +26,6 @@
...
+    "quality": "verified"
```
Comment (mal-tee) [inline companies/abis.json:30]:
```suggestion
    "quality": "tested"
```

**Review by mal-tee — COMMENTED** [review]
So, should we update the record like this for the time being?

**Review by mal-tee — APPROVED** [review]
Thanks!

Should we create an issue for the discussion regarding @rugk 's complaint or do we just use this PR? :D

**Comment by rugk** [comment]
Problem solved.

LDI Hessen:
>  Die ABIS GmbH hat mir nunmehr bestätigt, dass der Prozess zur Auskunftserteilung den datenschutzrechtlichen Maßgaben angepasst worden ist.
>
>Die Betroffenenrechte erfordern keine bestimmte Form, insbesondere keine Unterschrift der betroffenen Person. Eine eindeutige Identifikation des Betroffenen ist mit einer Unterschrift auch gar nicht möglich. Nach Art. 12 Abs. 6 DS-GVO kann der Verantwortliche lediglich dann zusätzliche Informationen anfordern, welche zur Bestätigung der Identität der betroffenen Person erforderlich sind, wenn er begründete Zweifel an der Identität der natürlichen Person hat. Ein voraussetzungsloses Anfordern weiterer Daten ist damit nicht vereinbar. Eine Unterschrift der betroffenen Person wird zukünftig von der ABIS GmbH nicht mehr verlangt.
>
>Die Erteilung von Auskunftsersuchen muss zudem auf verschiedenen Kommunikationskanälen geschehen. Entsprechend erfolgt eine Auskunftserteilung durch die ABIS GmbH künftig nicht mehr nur mittels Briefpost.
>
>Ich bedanke mich für Ihren Hinweis auf das nicht datenschutzkonforme Verhalten der ABIS GmbH, welcher zu einer Optimierung des Datenschutzes beigetragen hat, und verbleibe[…]

**Comment by mal-tee** [comment]
Wow, that is great!

**Comment by baltpeter** [comment]
@rugk I'm a little late to the party but that is indeed great. Thanks for submitting the complaint!

I think it would be good to have a blog post on this for our website (this PR is kind of hard to find). That post could serve two important purposes: a) document that requiring signatures for requests is not OK and b) be an awesome (and unfortunately quite rare, thus far) example of a successful complaint that actually changed the situation for the better for everyone.

Would you be OK with sharing a redacted copy of your correspondence with the LDI Hessen with me? That would make writing the post a lot easier. You can reach me via [email](mailto:benni@datenanfragen.de) ([PGP key](https://www.datenanfragen.de/pgp/AF1381BD.asc)) or [Matrix](https://matrix.to/#/@benni:matrix.altpeter.me)).

But if you don't want that, that's obviously totally OK as well.

**Comment by rugk** [comment]
Done. (ID: `GqLAkqcDekj,iH6,3oCx`)

**Comment by rugk** [comment]
Or, of course [you can FOI-request that communication from the LfDI Hessen](https://fragdenstaat.de/). 
Agency reference: AZ 90.20.77:0245

**Comment by baltpeter** [comment]
Thank you! I have opened #524 for the post.

> Or, of course you can FOI-request that communication from the LfDI Hessen.

Good point! As someone from Lower Saxony that isn't a right I am used to having. :D

**Comment by mal-tee** [comment]
https://www.datenanfragen.de/blog/abis-unterschrift-dsgvo-anfragen/
The article is live 🥳

**Pull request merged**